packages/components/src/components/Drawer/DrawerHeader.tsx

```diff
@@ -1,23 +1,30 @@
 import React from "react";
 import { DialogDismiss } from "ariakit/dialog";
+import type { SystemProp, Theme } from "@xstyled/styled-components";
 import { Box } from "../../primitives/Box";
 import { Button } from "../Button";
 
 export interface DrawerHeaderProps
   extends Omit<React.HTMLAttributes<HTMLDivElement>, "color"> {
   /** The contents of the drawer header */
   children: NonNullable<React.ReactNode>;
+  marginBottom?: SystemProp<keyof Theme["space"], Theme>;
   padding?: "space0" | "space60";
 }
 
 /** The header content area of the drawer. */
 const DrawerHeader = React.forwardRef<HTMLDivElement, DrawerHeaderProps>(
-  ({ children, padding = "space60", ...props }, ref) => {
+  (
+    { children, marginBottom = "space0", padding = "space60", ...props },
+    ref,
+  ) => {
     return (
       <Box.div
+        alignItems="center"
         display="flex"
         gap="space30"
         justifyContent="space-between"
+        marginBottom={marginBottom}
```

Comment:
Why do we need a marginBottom? I ran storybook and noticed that it's showing ok for what we need, no? I mean, padding should be enough. Also, we don't set marginBottom on Figma.

![image](https://github.com/Localitos/pluto/assets/5320963/30730a2a-1a50-432d-8066-6b151f391a43)

Reply:
I get your point completely, this is just to provide more flexibility to this component for future use.
If you think this specific change should be deleted I'm fine with it, just let me know.

Reply:
For me, with the changes you did on this [PR](https://github.com/Localitos/pluto/pull/1490), we already have a padding of 16px that it's the same what we have on Figma. So, it seems to me that we don't need to introduce anything related to spaces on this PR.